test/e2e/tests/confirmations/transactions/contract-interaction-redesign.spec.ts

```diff
@@ -212,7 +226,10 @@ describe('Confirmation Redesign Contract Interaction Component', function () {
           fixtures: new FixtureBuilder({ inputChainId: CHAIN_IDS.OPTIMISM })
             .withPermissionControllerConnectedToTestDapp()
             .withPreferencesController({
-              preferences: { redesignedConfirmationsEnabled: true },
+              preferences: {
+                redesignedConfirmationsEnabled: true,
+                isRedesignedConfirmationsDeveloperEnabled: true,
+              },
```

Comment:
this file does not need to set an extra set of preferences that the original has. The original changes 4 more lines due to differences in the tests between Version v12.2.0 and develop ([compare](https://github.com/MetaMask/metamask-extension/pull/25520/files#diff-9b7dc3b211ccc46387c285548ff2177a7f4f499a74715bba074069f0c7f82222L26))

+1 (line change in above comment) - 4 not added here = -3 added lines changed. This is how we have `+469 -266` diff here and `+472 -266` in the original PR

Reply:
explaining how the line diff makes sense here 👍🏼  LGTM!